js/chessMoveReminder.js

```diff
@@ -27,71 +26,84 @@ function parseSecondsFromClock(clock) {
 
 function chessMoveReminder() {
     let timer;
-    let seconds = 5;
     let audio;
 
-    let playAudio = async function(who) {
-        let audioUrl = 'audio/SoundEffect/SoundEffect.mp3';
-        if (who === 'HikaruGotham') {
-            audioUrl = 'audio/HikaruGotham/' + randomPositiveNumberWithoutRepeat(29) + '.wav';
+    let playAudio = async function (who) {
+        let audioUrl = "audio/SoundEffect/SoundEffect.mp3";
```

Comment:
Please leave the quotes as single quotes: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

(Here and all future instances)